public/javascripts/redirect.js

```diff
@@ -0,0 +1,74 @@
+// Redirect all the existing URLs with fragment identifiers in to the new section
+(function () {
+  'use strict';
+
+
+  var getNewRouteFor = function (fragment) {
+    var baseURL = 'http://govuk-elements.herokuapp.com';
```

Comment:
`window.location.host` misses out the protocol. My suggestion is a bit hackier but yeah, using `window.location.protocol + '//' + `window.location.host` is probably more correct.